geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,73 @@
+import geomstats.geometry.spd_matrices as spd
+import geomstats.backend as gs
+
+class LogNormal:
+    """ LogNormal Sampler 
+
+    Parameters:
+    ----------
+    manifold: str, {\'Euclidean\', \'SPDmanifold\'}
+        Name of the Manifold
+    mean: array-like, shape=[..., dim]
+        Mean of the Distribution
+    cov: array-like, shape=[..., dim]
+        Covariance of the Distribution. Should be Positive Semi Definite
+    Returns:
+    --------
+    samples
+
+    Examples:
+    --------
+
+    References:
+    ----------
+    Lognormal Distributions and Geometric Averages of Symmetric Positive Definite Matrices
+    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5222531/
+    """
+    def __init__(self,manifold,mean,cov=None):
```

Comment:
The code is not PEP8 compliant, e.g. there should be spaces after the ",", etc... Maybe DeepSource is catching this? Otherwise, it could be good to run flake8 to check.